clancy_database/models.py

```diff
@@ -76,6 +80,23 @@ def to_dict(self):
             data["aspect_pair"] = self.get_aspect_pair()
         return data
 
+    def get_icons(self):
+        found = ""
+        file =  os.path.join(ROOT_DIR, 'clancy_database/data/icons.csv')
+        with open(file, encoding='utf-8-sig') as csvf:
```

Comment:
I think this approach is fine for initial development and testing, but I think we'll ultimately want to load this data into the DB so that we can query it along with other lemma data. As it stands, the CSV file is read and parsed for each lemma, which isn't going to scale well. We could improve that by doing the CSV file reading/parsing once when the module is loaded, but we still have the problem of integrating with the other lemma data and being able to query it consistently.

For reference, here's one way to go about integrating it with the DB:

1. Update the `lemma` table and add columns `icon_url, icon_license, icon_attribution`. This table is defined in [schema.sql](https://github.com/Harvard-ATG/visualizing_russian_tools/blob/b6a65ac25dd884b0bfc18551f3ccae7ec22a5f03/clancy_database/spreadsheet/schema.sql) and gets created as part of the [spreadsheet import process](https://github.com/Harvard-ATG/visualizing_russian_tools/tree/b6a65ac25dd884b0bfc18551f3ccae7ec22a5f03#source-data).
2. Update the `Lemma` model so that it knows about those new columns.
3. Add a management command that processes `icons.csv` and updates the database (e.q. query the `Lemma` model to find a matching lemma and update the icon for it, similar to the logic that is there now).

Reply:
Ahh ok! Makes total sense! Would that combine these two stories into one? [ATGU-3266](https://jira.huit.harvard.edu/browse/ATGU-3266) and [ATGU-3264](https://jira.huit.harvard.edu/browse/ATGU-3264)

Reply:
@Tanvez Yes, it definitely would! Note that there might be some other details around updating the DB that I'm forgetting at the moment, so if you get stuck let me know.